### PR TITLE
add 6.1.0 release notes

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran.alex@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add 6.1.0 release notes

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9702
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.1.0.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+    - Fix the issue that the Plan Cache might be wrong on the PointGet plan [#3237](https://github.com/pingcap/tidb/issues/3237)
+    - Fix the issue that query result might be wrong when Plan Cache is started in the RC isolation level [#34447](https://github.com/pingcap/tidb/issues/34447)
+
++ TiKV
+
+    - Fix the issue that the Raft log lag is increasing when a TiKV instance is taken offline [#12161](https://github.com/tikv/tikv/issues/12161)
+    - Fix the issue that TiKV panics and destroys peers unexpectedly because the target Region to be merged is invalid [#12232](https://github.com/tikv/tikv/issues/12232)
+    - Fix the issue that TiKV reports the `failed to load_latest_options` error when upgrading from v5.3.1 or v5.4.0 to v6.0.0 [#12269](https://github.com/tikv/tikv/issues/12269)
+    - Fix the issue of OOM caused by appending Raft logs when the memory resource is insufficient [#11379](https://github.com/tikv/tikv/issues/11379)
+    - Fix the issue ofTiKV panic caused by the race between destroying peers and batch splitting Regions [#12368](https://github.com/tikv/tikv/issues/12368)
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:394]:
```suggestion
    - Fix the issue of TiKV panic caused by the race between destroying peers and batch splitting Regions [#12368](https://github.com/tikv/tikv/issues/12368)
```

File: releases/release-6.1.0.md [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+
+* Support killing queries or connections globally
+
+    You can control the Global Kill feature using the `enable-global-kill` configuration (enabled by default).
+
+    Before TiDB v6.1.0, when an operation consumes a lot of resources and causes cluster stability issues, you have to connect to the target TiDB instance and then run the `KILL TIDB ${id};` command to terminate the target connection and operation. In the case of many TiDB instances, this method is not easy to use and prone to wrong operations. Starting from v6.1.0, the `enable-global-kill` configuration is introduced and enabled by default. You can run the kill command in any TiDB instance to terminate a specified connection and operation, without worrying about incorrectly terminating other queries or sessions by mistake when there is a proxy between the client and TiDB. Currently, TiDB does not support using Ctrl+C to terminate queries or sessions.
+
+    [User document](/tidb-configuration-file.md#enable-global-kill-introduced-from-v610-version), [#8854](https://github.com/pingcap/tidb/issues/8854)
+
+* TiKV API v2
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:177]:
```suggestion
* TiKV API V2
```

File: releases/release-6.1.0.md [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+
+    Before v6.1.0, when TiKV is used as Raw Key Value storage, TiKV only provides basic Key Value read and write capability because it only stores the raw data passed in by the client.
+
+    TiKV API v2 provides a new Raw Key Value storage format and access interface, including:
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:181]:
```suggestion
    TiKV API V2 provides a new Raw Key Value storage format and access interface, including:
```

File: releases/release-6.1.0.md [hunk 4]
```diff
@@ -0,0 +1,440 @@
...
+
+    * The data is stored in MVCC (Multi Version Concurrency Control) and the change timestamp of the data is recorded. This feature will lay the foundation for implementing Change Data Capture and incremental backup and restore.
+    * Data is scoped according to different usage and supports co-existence of a single TiDB cluster / Transactional KV / RawKV applications.
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:184]:
```suggestion
    * Data is scoped according to different usage and supports co-existence of a single TiDB cluster, Transactional KV, RawKV applications.
```

File: releases/release-6.1.0.md [hunk 5]
```diff
@@ -0,0 +1,440 @@
...
+
+    Note: Due to significant changes in the underlying storage format, after enabling API v2, you cannot roll back a TiKV cluster to a version earlier than v6.1.0. Downgrading TiKV might result in data corruption..
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:186]:
```suggestion
    Note: Due to significant changes in the underlying storage format, after enabling API V2, you cannot roll back a TiKV cluster to a version earlier than v6.1.0. Downgrading TiKV might result in data corruption.
```

File: releases/release-6.1.0.md [hunk 6]
```diff
@@ -0,0 +1,440 @@
...
+| TiDB | [`enable-stats-cache-mem-quota`](/tidb-configuration-file.md#enable-stats-cache-mem-quota-new-in-v610) | Newly added | Controls whether to enable the memory quota for the statistics cache. |
+| TiKV | [`raft-engine.enable`](/tikv-configuration-file.md#enable-1) | Modified | The default value is changed from `FALSE` to `TRUE`. |
+| TiKV | [`region-max-keys`](/tikv-configuration-file.md#region-max-keys) | Modified | The default value is changed from 1440000 to `region-split-keys / 2 * 3`. |
+| TiKV | [`region-max-size`](/tikv-configuration-file.md#region-max-size) | Modified | The default value is changed from 144 MB to `region-split-size / 2 * 3`. |
+| TiKV | [`causal-ts.renew-batch-min-size`](/tikv-configuration-file.md#renew-batch-min-size) | Newly added | The minimum number of locally cached timestamps. |
```
Comment (Oreoxmt) [inline releases/release-6.1.0.md:277]:
```suggestion
| TiKV | [`coprocessor.enable-region-bucket`](/tikv-configuration-file.md#enable-region-bucket-new-in-v610) | Newly added | Determines whether to divide a Region into smaller ranges called buckets. |
| TiKV | [`coprocessor.region-bucket-size`](/tikv-configuration-file.md#region-bucket-size-new-in-v610) | Newly added | The size of a bucket when `enable-region-bucket` is true. |
| TiKV | [`causal-ts.renew-batch-min-size`](/tikv-configuration-file.md#renew-batch-min-size) | Newly added | The minimum number of locally cached timestamps. |
```

File: releases/release-6.1.0.md [hunk 7]
```diff
@@ -0,0 +1,428 @@
...
+| TiKV | [`causal-ts.renew-interval`](/tikv-configuration-file.md#renew-interval) | Newly added | The interval at which the locally cached timestamps are refreshed. |
+| TiKV | [`max-snapshot-file-raw-size`](/tikv-configuration-file.md#max-snapshot-file-raw-size-new-in-v610) | Newly added | The snapshot file will split to multiple files when the snapshot file size exceeds this value. |
+| TiKV | [`raft-engine.memory-limit`](/tikv-configuration-file.md#memory-limit) | Newly added | Specifies the limit on the memory usage of Raft Engine. |
+| TiKV | [`storage.background-error-recovery-window`](/tikv-configuration-file.md#background-error-recovery-window-new-in-v610) | Newly added | The maximum recovery time is allowed after RocksDB detects a recoverable background error. |
+| TiKV | [`storage.block-cache.api-version`](/tikv-configuration-file.md#api-version-new-in-v610) | Newly added | The storage format and interface version used by TiKV when TiKV serves as the raw key-value store. |
```
Comment (pingyu) [inline releases/release-6.1.0.md:273]:
```suggestion
| TiKV | [`storage.api-version`](/tikv-configuration-file.md#api-version-new-in-v610) | Newly added | The storage format and interface version used by TiKV when TiKV serves as the raw key-value store. |
```
The `tikv-configuration-file.md` is wrong. See https://github.com/pingcap/docs/pull/8844.

**Review by TomShawn — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**